src/runtime/EnvironmentRecord.h

```diff
@@ -216,6 +219,10 @@ class ObjectEnvironmentRecord : public EnvironmentRecord {
     {
         auto result = m_bindingObject->get(state, ObjectPropertyName(name));
         if (result.hasValue()) {
+            Value unscopables = m_bindingObject->get(state, ObjectPropertyName(state, state.context()->vmInstance()->globalSymbols().unscopables)).value(state, m_bindingObject);
+            if (unscopables.isObject() == true && unscopables.asObject()->get(state, ObjectPropertyName(state, name)).value(state, unscopables).toBoolean(state) == true) {
+                return GetBindingValueResult();
```

Comment:
`@@unscopables` is rarely used, so how about adding `UNLIKELY` macro for this if statement?

Reply:
#296 has landed, so please update the coding style according to it.